.fixtures.yml

```diff
@@ -1,4 +1,6 @@
 fixtures:
   repositories:
     archive: https://github.com/voxpupuli/puppet-archive.git
+    file_capability: https://github.com/smoeding/puppet-file_capability.git
     stdlib: https://github.com/puppetlabs/puppetlabs-stdlib.git
+    
```

Comment:
there is an empty newline but without an \n?

Reply:
Solving merge conflicts at the iPad sucks. ;)